src/Nox.Ui.Blazor.Lib/Components/NoxTypes/ViewArea.razor.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.AspNetCore.Components;
+using Nox.Types;
+using Nox.Ui.Blazor.Lib.Models;
+
+namespace Nox.Ui.Blazor.Lib.Components.NoxTypes;
+
+public partial class ViewArea : ComponentBase
+{
+
+    #region Declarations
+
+    [Parameter]
+    public Decimal? Area { get; set; }
+
+    [Parameter]
+    public AreaUnit AreaUnit { get; set; } = AreaUnit.SquareMeter;
+
+    [Parameter]
+    public string Format { get; set; } = "#,##0.##";
+
+    public string DisplayArea
+    {
+
+        get
+        {
+            if (Area != null)
+            {
+                switch (Area)
+                {
+                    case < 0:
+                        float rtnMinusArea = 0;
+                        return rtnMinusArea.ToString(Format) + DisplayAreaUnit();
```

Comment:
what should be the logic for Area<0? As I understand here if Area<0 we display "0 sq ft" in the required Format. Is it correct? Why isn't it just empty string if we consider it as a "wrong" value?

Reply:
on latest commit